proposals/lip-0053.md

```diff
@@ -483,7 +603,7 @@ def verify(trs: Transaction) -> None:
         verifyCertificate(ccu)
     
     # Liveness condition is only checked on the mainchain for the first CCU with a non-empty inbox update.
-    if chainAccount(ccu.params.sendingChainID).status == CHAIN_STATUS_REGISTERED and len(ccu.params.inboxUpdate) > 0:
+    if chainAccount(ccu.params.sendingChainID).status == CHAIN_STATUS_REGISTERED and ccu.params.inboxUpdate is not empty:
         verifyLivenessConditionForRegisteredChains(ccu)
```

Comment:
This check is now also done on sidechain, which should not be the case, right?

Reply:
nice catch, I will edit